extensions/ql-vscode/src/data-extensions-editor/data-extensions-editor-view.ts

```diff
@@ -211,6 +212,13 @@ export class DataExtensionsEditorView extends AbstractWebview<
 
         await Promise.all([this.setViewState(), this.loadExternalApiUsages()]);
 
+        break;
+      case "hideModeledApis":
+        await this.updateModelDetailsPanelState(
```

Comment:
We'll need to update the other places we call `this.updateModelDetailsPanelState` to also pass the new `hideModeledApis` value there. For example the one in `onDidChangeViewState` which triggers when we change which data extensions editor is active, but it would be safer to always pass the values if possible.

I recommend storing the `hideModeledApis` value as a field on `DataExtensionsEditorView`, in the same way as we do for `externalApiUsages`.